cockroachcloud/export-logs.md

```diff
@@ -199,11 +199,11 @@ Perform the following steps to enable log export from your {{ site.data.products
             - Sends log entries in the `OPS`, `HEALTH`, and `STORAGE` [logging channels](/docs/{{site.current_cloud_version}}/logging-overview.html#logging-channels) to an AWS CloudWatch log group named `devops`, but only for those entries that are of log [severity level](/docs/{{site.current_cloud_version}}/logging.html#logging-levels-severities) `WARNING` or higher.
```

Comment:
text here will need to be amended to add the `crdb-*` prefix to log group names.

Reply:
Also amend the next two substeps to also include the updated three renamed `log_name` values, and also for GCP.